app/models/trainee.rb

```diff
@@ -158,6 +158,8 @@ class Trainee < ApplicationRecord
     end
   end
 
+  COMPLETE_STATES = %w[recommended_for_award withdrawn awarded].freeze
```

Comment:
have called these `COMPLETE_STATES` as we are considering them complete regardless of the data we have - not sure if there is a better name for this constant?